definitions/npm/react-datepicker_v1.4.x/flow_v0.84.x-/react-datepicker_v1.4.x.js

```diff
@@ -0,0 +1,88 @@
+// Based on https://github.com/Hacker0x01/react-datepicker/blob/master/docs/datepicker.md
+
+declare module 'react-datepicker' {
+  declare type moment$Moment = Object;
```

Comment:
Is there a way to reference the actual `moment$Moment` definition from the repo? I've done what rsuite_v3... does but aliasing `Moment` to `Object` is obviously pretty bad.

Reply:
/cc @villesau

Reply:
Ah... thanks @villesau, I'll read through those and subscribe